src/main/java/com/openmrs/migrator/core/services/impl/PDIServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package com.openmrs.migrator.core.services.impl;
+
+import com.openmrs.migrator.core.services.PDIService;
+import java.io.InputStream;
+import org.pentaho.di.core.KettleEnvironment;
+import org.pentaho.di.core.Result;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.logging.LogLevel;
+import org.pentaho.di.trans.Trans;
+import org.pentaho.di.trans.TransMeta;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PDIServiceImpl implements PDIService {
+
+  private static Logger LOG = LoggerFactory.getLogger(PDIServiceImpl.class);
+
+  @Override
+  public void runTransformation(InputStream xmlStream) throws KettleException {
```

Comment:
We need tests for this method

Reply:
Sure. I think it will have to be an integration test because I don't think its possible to mock PDI using  Mockito.

Reply:
yes sure, an integration test would be better

Reply:
still needs to be done @cfaife